### DISCUSSION TITLE
Interact with Chrome extension using Seleniumbase

### DISCUSSION THREAD

**Discussion by Remi13Git** [seleniumbase/SeleniumBase#2686] [Q&A] [answered]
I would like to interact with a Chrome extension (Metamask) using Seleniumbase. The problem is that SB can't detect the elements present on the page. It manages to position itself on the extension page + get the activ url but when it comes to interacting with the elements, it don't find them.

I've unzipped the extension correctly, it opens correctly and I can interact with it manually.

I have exactly the same problem with undetected_chromedriver. However, Metamask automation works using Selenium. I think the conflict comes from the privacy added by UC and SB.

Do you have any solutions for unlocking extension elements accessibility for Seleniumbase?

If not, any ideas for automating Metamask in headless mode (without using pyautogui/pynput or keyboard shortcuts, as this requires a graphical interface)?

Here's my code:
`from seleniumbase import SB

with SB(
        demo=True,
        extension_dir="temp/metamask-chrome-11.13.1"
        ) as sb:
        sb.open('about:blank')
        sb.sleep(3)
        sb.switch_to_newest_window()
        activ_url = sb.get_current_url()
        print(activ_url)
        sb.click('#onboarding__terms-checkbox') `

What I tried : 
-use others selector (xpath, css selector) 
-wait more time to load page 
-search an iframe to switch on 
-disable features on SB (undetected=False, remote_debug=False ...) 
-sb.press_keys('body', '\t') to moove with tab but don't find body element

I don't want to go back to Selenium classic 🥲🥲

**Comment by mdmintz** [comment, marked as answer]
Selenium cannot easily interact with Extension controls that aren't located directly inside the web page being automated.

You might be able to find answers on https://stackoverflow.com/
